cob_hardware_config/cob3-2/config/robot_modules.yaml

```diff
@@ -0,0 +1 @@
+robot_modules: ['base','tray','torso','head','arm','sdh']
```

Comment:
does robot_modules.yaml belong to cob_hardware_config or cob_default_robot_config or both? see my comment above....